### ISSUE TITLE
toHaveBeenCalledWith creates confusing output when used with single number argument

### ISSUE THREAD

**Issue by SanityResort** [jasmine/jasmine#913]
Let's say you have a mock with a function like
```
var myMock = { doStuff = jasmine.createSpy('myMock.doStuff')}
```
and put that into a service that calls:
```
myMock.doStuff(2)
```
and then expect
```
expect(myMock.doStuff).toHaveBeenCalledWith(10)
```
jasmine creates the following error message:
```
Expected spy timeout.cancel to have been called with [ <circular reference: Object> ] but actual calls were [ <circular reference: Object> ].
```
If you use strings or more than one number argument everything gets printed fine. If the mock gets called with the expected value the test passes, so it seems to be just a formatting issue with the failure message.

**Comment by slackersoft** [comment]
I think this is actually a PrettyPrinter issue, and it should have been fixed in 2.3.0. What version are you using? Can you upgrade to get the fix?

**Comment by SanityResort** [comment]
We had this issue by using the jasmine sbt plugin: https://github.com/joescii/sbt-jasmine-plugin

I just checked the require code and it seems the plugin relies on jasmine 2.0.3 so maybe that is the problem there.

I will create an issue there, maybe they can upgrade their dependency.
